com.unity.render-pipelines.high-definition/Runtime/Material/BuiltinUtilities.hlsl

```diff
@@ -14,6 +14,13 @@ float2 CalculateMotionVector(float4 positionCS, float4 previousPositionCS)
     previousPositionCS.xy = previousPositionCS.xy / previousPositionCS.w;
 
     float2 motionVec = (positionCS.xy - previousPositionCS.xy);
+
+#ifdef KILL_MICRO_MOVEMENT
+    // We multiply by 2 as the motion vectors have been already converted to NDC when we reach this function.
```

Comment:
where is the multiply by 2 mention in the comment?

Reply:
Oh because I moved the code and forgot to remove the comment, will do remove that